### PR TITLE
ushift: Add 4.13 to testgrid. Rework 4.12 configuration

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-release-4.12-nightly-conformance-serial

File: ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml [hunk 1]
```diff
@@ -1,193 +0,0 @@
...
-    workflow: openshift-microshift-e2e-openshift-conformance-sig-storage
-- as: e2e-reboot
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    workflow: openshift-microshift-e2e-reboot-tests
-- as: rebase-on-nightlies
-  cron: 0 5 * * *
-  steps:
-    workflow: openshift-microshift-rebase
```
Comment (pmtk) [inline ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml:188]: I think we might want that

Comment (pacevedom) [reply ci-operator/config/openshift/microshift/openshift-microshift-main__periodics.yaml:188]: Oops you are right. We also need this on 4.12, right?

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-release-4.12-nightly-conformance-serial

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-release-4.12-nightly-conformance-serial

**Comment by pacevedom** [comment]
/pj-rehearse periodic-ci-openshift-microshift-release-4.12-nightly-conformance-serial

**Comment by pacevedom** [comment]
@ingvagabund @sosiouxme PTAL

**Comment by pacevedom** [comment]
/uncc @ingvagabund 
My bad.

**Pull request closed without merging**